src/VRCFaceTracking.Avalonia/Controls/SettingsToggle.axaml

```diff
@@ -5,16 +5,23 @@
     x:Class="VRCFaceTracking.Avalonia.Controls.SettingsToggle"
     xmlns="https://github.com/avaloniaui"
     xmlns:controls="clr-namespace:VRCFaceTracking.Avalonia.Controls"
+    xmlns:converters="clr-namespace:VRCFaceTracking.Avalonia.Converters"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
     <Border
         Background="{DynamicResource SystemChromeMediumLowColor}"
         CornerRadius="6"
         Padding="15,5,15,5">
-        <Grid ColumnDefinitions="Auto,*,Auto" VerticalAlignment="Center">
+        <Grid ColumnDefinitions="Auto, Auto,*,Auto" VerticalAlignment="Center">
+          <PathIcon Data="{Binding IconPath, RelativeSource={RelativeSource AncestorType=controls:SettingsToggle}, Converter={x:Static converters:TypeConverters.IconConverter}}"
+                    Grid.Column="0"
+                    Background="#00000000"
```

Comment:
You should be able to remove a fully transparent background from a PathIcon, and it won't render.